### PR TITLE
Update the index page description and add a GIF

### PR DESCRIPTION
**Pull request by GalOshri**
Update the TensorBoard docs index page:
- Description includes brief itemization of some of TensorBoard's capabilities
- GIF shows some of TensorBoard's capabilities
- Adding a link to a Google AI blog on TensorBoard's WIT plugin

### CODE REVIEW

**Comment by lamberta** [comment]
@MarkDaoust 
There's a ~2mb limit for images before we need to host them directly in tensorflow.org. Mark to confirm

**Comment by MarkDaoust** [comment]
Most copybara configs have 2Mb set as a limit.
But AFAIK it's just good practice for keeping your repo-size down.

You can submit it (internally) to site-tensorflow/en/images/tensorboard if you want to avoid ballooning the repo.

**Comment by nfelt** [comment]
Justine had some tips for compressing a video file for gif generation back on this old PR: https://github.com/tensorflow/tensorboard/pull/694 (which unfortunately I don't think was ever properly followed up on).

Beyond reducing framerate we might also be able to downsample the color palette, which for non-photographic stuff is generally not going to be very noticeable anyway.

**Comment by GalOshri** [comment]
Thank you for catching this. How do people feel about a 500 KB GIF?

**Comment by nfelt** [comment]
500k seems fine to me personally.  It's certainly below the 2MB limit.

**Comment by stephanwlee** [comment]
> Most copybara configs have 2Mb set as a limit.

But it says 2Mb not 2MB.

**Comment by nfelt** [comment]
I think we're all talking about megabytes here.

**Comment by MarkDaoust** [comment]
Yeah, that was a typo.  MB.

**Comment by GalOshri** [comment]
I updated the GIF.

> Yeah, that was a typo. MB.

Great pun if intended.

File: docs/_index.yaml [hunk 1]
```diff
@@ -8,22 +8,36 @@ landing_page:
...
+          <li>Viewing histograms and distributions of Tensors (e.g. weights 
+          and biases) to track how they change over time</li>
+          <li>Projecting embeddings to a lower dimensional space</li>
+          <li>Displaying images, text, and audio data</li>
+          <li>Profiling TensorFlow programs</li>
+          <li>And much more</li>
+        </ul>
 
-      image_path: /images/graph_vis_animation.gif
+      image_path: /images/tensorboard.gif
```
Comment (wchargin) [inline docs/_index.yaml]: We use `/images/*` here but `/resources/images/*` below; is this right?
(I see the `/resources/images/*` files within google3 but I don’t see
where they come from or how our `./images` might be merged into the same
path.)

File: docs/_index.yaml [hunk 2]
```diff
@@ -8,22 +8,36 @@ landing_page:
...
-        The computations you'll use TensorFlow for—like training a massive deep
-        neural network—can be complex and confusing. To make it easier to
-        understand, debug, and optimize TensorFlow programs, we've included a
-        suite of visualization tools called TensorBoard.
+        TensorBoard provides the visualization and tooling needed for 
+        machine learning experimentation. It enables:
+        <ul style="padding-left: 20px; margin: 20px 0;">
+          <li>Tracking and visualizing metrics such as loss and accuracy</li>
+          <li>Visualizing the model graph (ops and layers)</li>
+          <li>Viewing histograms and distributions of Tensors (e.g. weights
```
Comment (wchargin) [inline docs/_index.yaml]: nit: Add comma after *e.g.*.

File: docs/_index.yaml (hunk 1 above)
Comment (GalOshri) [reply docs/_index.yaml]: I am not sure. I think the references to `/resources/images/*` were in the original addition from  @lamberta. I'm happy to move the GIF to another folder if it makes sense. Do the other images need to be moved to this repo or would they be added as part of the website publishing?

Comment (lamberta) [reply docs/_index.yaml]: For images you want to keep within your site, creating a local `./images` directory is fine and use a relative path to include it.
For large files, we need to put them in our main site images directory in g3. We can create an `$SITE_ROOT/images/tensorboard/...` directory for you

Comment (GalOshri) [reply docs/_index.yaml]: Per offline conversation, leaving the resource card image as is for now. We'll update this a bit later.

File: docs/_index.yaml (hunk 2 above)
Comment (GalOshri) [reply docs/_index.yaml]: Never knew this. Updated.

**Pull request merged**